modules/lang-painless/src/main/java/org/elasticsearch/painless/Definition.java

```diff
@@ -770,10 +770,10 @@ public Definition(List<Whitelist> whitelists) {
         }
 
         // precompute runtime classes
-        for (String painlessStructName : structsMap.keySet()) {
-            Struct painlessStruct = structsMap.get(painlessStructName);
+        for (Map.Entry<String, Struct> entry : structsMap.entrySet()) {
+            Struct painlessStruct = structsMap.get(entry.getKey());
```

Comment:
Again, this doesn't seem to actually use the `entry.getValue()`.

Reply:
I fixed it.